### PR TITLE
Add delete comment feature given the shows id.

### PR DESCRIPTION
**Pull request by josefj1519**
Assignee: LiRq95

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DeleteDataServlet.java [hunk 1]
```diff
@@ -30,9 +31,11 @@
...
-    Key taskEntityKey = KeyFactory.createKey("Shows", id);
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-    datastore.delete(taskEntityKey);
+    Query query = new Query("Shows");
+    PreparedQuery results = datastore.prepare(query);
+    for (Entity entity : results.asIterable()) {
+      datastore.delete(entity.getKey());
+    }
   }
 }
```
Comment (LiRq95) [inline portfolio/src/main/java/com/google/sps/servlets/DeleteDataServlet.java:41]: Remember to keep newlines at end of file (can optionally add in this review or following reviews)

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -42,10 +42,7 @@ async function getShowTitles(maxComments){
...
-
-function deleteShow(id) {
-  const params = new URLSearchParams();
-  params.append('id', id);
-  fetch('/delete-data', {method: 'POST', body: params});
-  getShowTitles(5);
+ async function deleteShow() {
+  await fetch('/delete-data', {method: 'POST'});
+  await getShowTitles(0);
 }
```
Comment (LiRq95) [inline portfolio/src/main/webapp/script.js:48]: EoF newline

Comment (josefj1519) [reply portfolio/src/main/webapp/script.js]: Fix.

File: portfolio/src/main/java/com/google/sps/servlets/DeleteDataServlet.java (hunk 1 above)
Comment (josefj1519) [reply portfolio/src/main/java/com/google/sps/servlets/DeleteDataServlet.java]: Fix.

**Pull request merged**